### PR TITLE
146381 - Add YCS as a user that can Manage Academy Openings

### PR DESCRIPTION
**Pull request by Kola-Abolarinwa**
# Description
As a Youth Custody Service (YCS) or back Office (BO) user of GIAS, I want the ability manage academy openings for establishments with an establishment type of Secure academy 16-19, so that I can control when the establishments I own open and edit academy names. 

## Type of change
- [X] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
Login as a Youth Custody Service (YCS) user. Click on the 'Tools' link. You should now be able to see the '[Manage academy openings](https://localhost:44309/Establishments/manage/academy-openings)' link and it should allow you use/click on the link and it should take you to the 'Manage academy openings' page. 
Please note that this ticket is not concerned with ensuring it functions as expected.
- [X] Manually Tested by Dev [@Kola-Abolarinwa ]

## Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code

### CODE REVIEW

**Review by anthony9853 — APPROVED** [review]

File: Web/Edubase.Web.UI/Helpers/AuthorizedRoles.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ public static class AuthorizedRoles
         public const string CanBulkUpdateEstablishments = EdubaseRoles.EDUBASE + "," + EdubaseRoles.EDUBASE_CMT + "," + EdubaseRoles.AP_AOS + "," + EdubaseRoles.APT + "," + EdubaseRoles.EDUBASE_CHILDRENS_CENTRE_POLICY + "," + EdubaseRoles.EFADO + "," + EdubaseRoles.EFAHNS + "," + EdubaseRoles.FST + "," + EdubaseRoles.IEBT + "," + EdubaseRoles.SOU + "," + EdubaseRoles.EDUBASE_LACCDO + "," + EdubaseRoles.LADO + "," + EdubaseRoles.LSU + "," + EdubaseRoles.UKRLP;
         public const string CanBulkUpdateGovernors = EdubaseRoles.ROLE_BACKOFFICE + "," + EdubaseRoles.ESTABLISHMENT + "," + EdubaseRoles.EDUBASE_GROUP_MAT + "," + EdubaseRoles.EFADO;
         public const string CanBulkCreateAcademies = EdubaseRoles.ROLE_BACKOFFICE + "," + EdubaseRoles.EFADO + "," + EdubaseRoles.AP_AOS;
-        public const string CanManageAcademyOpenings = EdubaseRoles.ROLE_BACKOFFICE + "," + EdubaseRoles.EFADO + "," + EdubaseRoles.AP_AOS;
+        public const string CanManageAcademyOpenings = EdubaseRoles.ROLE_BACKOFFICE + "," + EdubaseRoles.EFADO + "," + EdubaseRoles.AP_AOS+","+ EdubaseRoles.YCS;
```
Comment (CathLass) [inline Web/Edubase.Web.UI/Helpers/AuthorizedRoles.cs:16]: Reading the ticket, the requirement is to add a new link, not allow YCS to have access to the existing link:
GIVEN I'm a YCS or BO user of GIAS
WHEN selecting Tools
THEN I can view **Manage secure academy 16-19 openings**

Comment (anthony9853) [reply Web/Edubase.Web.UI/Helpers/AuthorizedRoles.cs:16]: Sorry I think you are right.  It does not really say that in plain english on the story but based on the fact its says "secure academy 16-19 openings" I assume it is a new link.

**Comment by Kola-Abolarinwa** [comment]
Further clarification showed this didn't capture all the required functionality.
